pkg/virt-handler/vm_test.go

```diff
@@ -1763,11 +1765,14 @@ var _ = Describe("VirtualMachineInstance", func() {
 				Expect(arg.(*v1.VirtualMachineInstance).Status.Interfaces[0].Name).To(Equal(podCacheInterface.Iface.Name))
 				Expect(arg.(*v1.VirtualMachineInstance).Status.Interfaces[0].IP).To(Equal(podCacheInterface.PodIP))
 				Expect(arg.(*v1.VirtualMachineInstance).Status.Interfaces[0].MAC).To(Equal(domain.Status.Interfaces[0].Mac))
-				Expect(arg.(*v1.VirtualMachineInstance).Status.Interfaces[0].IPs[0]).To(Equal(podCacheInterface.PodIP))
+				Expect(reflect.DeepEqual(arg.(*v1.VirtualMachineInstance).Status.Interfaces[0].IPs, podCacheInterface.PodIPs)).To(BeTrue())
 			}).Return(vmi, nil)
 
 			controller.Execute()
-		})
+		},
+			table.Entry("IPv4 only", []string{"2.2.2.2"}),
+			table.Entry("Dual stack", []string{"2.2.2.2", "fd10:244::8c4c"}),
```

Comment:
can we add another `table.Entry` with ipv6 as the primary? or is that not allowed?

Reply:
ipv6 primary isnt supported by k8s right now, and anyway the logic about creating the pod cache is mocked, so the unit test test just what it needed (there was a discussion about that in this PR)